src/docs/AppCanvas.vue

```diff
@@ -36,15 +36,24 @@ const initialConfig = {
   overflow: true,
   underline: false,
   strikethrough: false,
-
   // ❗️ IMPORTANT: always initialize with a system font (one that is globally available on all
   //  systems); any non-system fonts added MUST ALSO BE DOWNLOADED and won't render properly
   //  initially if not previously installed -- add non-system fonts to INDEX.HTML Google Font
   //  API request
   fontFamily: 'Times New Roman',
+  fontColor: '#708090',
+  underlineColor: '#0000ff',
+  strikethroughColor: '#ff0000',
+  underlineThickness: 1,
+  underlineOffset: 0,
+  strikethroughOffset: 0,
+  fontStyle: false,
```

Comment:
Given `fontStyle` is a string the formatting configuration, but here used as a boolean, it's a bit confusing. Can we rename this to `isItalic` so it's more obvious it's a boolean and not a string whose value matches acceptable values for the `fontStyle` format?